encrypted-media-respec.html

```diff
@@ -651,15 +699,15 @@
                 <dl class="switch">
                   <dt><a def-id="requirement-required"></a></dt>
                   <dd>
-                    <p>If the <var>implementation</var> does not support a persistent <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var>, return <code>null</code>.</p>
+                    <p>If the <var>implementation</var> does not support a persistent <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var> and <var>restrictions</var>, return <code>NotSupported</code>.</p>
                   </dd>
                   <dt><a def-id="requirement-optional"></a></dt>
                   <dd>
                     <p>Continue.<!-- Will be updated below. --></p>
                   </dd>
                   <dt><a def-id="requirement-not-allowed"></a></dt>
                   <dd>
-                    <p>If the <var>implementation</var> requires a <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var>, return <code>null</code>.</p>
```

Comment:
Unrelated to this CL, we should probably replace "Continue" with "Continue to the next step" to avoid any potential confusion with `continue` in programming languages.